Source/WebKit/DerivedSources-input.xcfilelist

```diff
@@ -1,4 +1,5 @@
 # This file is generated by the generate-xcfilelists script.
+$(BUILT_PRODUCTS_DIR)/DerivedSources/WebCore/HTTPHeaderNames.serialization.in
```

Comment:
I think this needs to use WEBCORE_PRIVATE_HEADERS_DIR to not break the internal build.